src/main/asciidoc/_chapters/developer.adoc

```diff
@@ -1801,6 +1801,58 @@ $ bin/hbase org.apache.hadoop.hbase.util.ChaosMonkey
 The output indicates that ChaosMonkey started the default `PeriodicRandomActionPolicy`
 policy, which is configured with all the available actions. It chose to run `RestartActiveMaster` and `RestartRandomRs` actions.
 
+==== ChaosMonkey without SSH
+
+Chaos monkey can be run without SSH using the Chaos service and ZNode cluster manager. HBase ships
+with many cluster managers, available in the `hbase-it/src/test/java/org/apache/hadoop/hbase/` directory.
+
+Set the following property in hbase configuration to switch to `ZNodeClusterManager`:
+`hbase.it.clustermanager.class=org.apache.hadoop.hbase.ZNodeClusterManager`
+
+Start chaos service on all hosts where you want to test chaos scenarios.
+
+[source,bash]
+----
+$ bin/hbase org.apache.hadoop.hbase.chaos.ChaosService
+
+Start chaos monkey from any one host.
```

Comment:
Change it to "Start chaos monkey runner from any host preferably from edge node"